spec/models/invoice_spec.rb

```diff
@@ -1,8 +1,40 @@
 require "rails_helper"
 
 RSpec.describe Invoice, type: :model do
-  it { should belong_to :customer }
-  it { should have_many :transactions }
-  it { should have_many :invoice_items }
-  it { should have_many(:items).through(:invoice_items) }
+  describe "relationships" do
+    it { should belong_to :customer }
+    it { should have_many :transactions }
+    it { should have_many :invoice_items }
+    it { should have_many(:items).through(:invoice_items) }
+    it { should have_many(:merchants).through(:items) }
+  end
+  describe "class methods" do
+    describe "#find_incomplete_invoices" do
+      it "returns all the invoices that have items that have not been shipped yet" do
+        merchant = create(:merchant)
+        customer = create(:customer)
+
+        item_1 = create(:item, merchant: merchant)
+        item_2 = create(:item, merchant: merchant)
+        item_3 = create(:item, merchant: merchant)
+        item_4 = create(:item, merchant: merchant)
+        item_5 = create(:item, merchant: merchant)
+        item_6 = create(:item, merchant: merchant)
+
+        @invoice_1 = create(:invoice, customer: customer) # 2 items that have not shipped
```

Comment:
super small detail here: because you're creating the invoice instances within the it block, you don't need the @ symbol for these instances (ln 24 -33) unless you want to pull them out into a before each block